core/payment-driver/dummy/src/service.rs

```diff
@@ -43,9 +55,10 @@ async fn init(_db: (), _caller: String, msg: Init) -> Result<Ack, GenericError>
     let mode = msg.mode();
 
     let msg = payment_srv::RegisterAccount {
-        platform: PLATFORM_NAME.to_string(),
         address,
         driver: DRIVER_NAME.to_string(),
+        network: "".to_string(),
```

Comment:
would it be nicer to have an `Option<String>` here instead of an empty one?

Reply:
That's actually a bug! I forgot to put `NETWORK_NAME` and `TOKEN_NAME` here.

Reply:
...and fixed!